lib/workos/sso.rb

```diff
@@ -124,6 +124,13 @@ def client
         @client
       end
 
+      sig { params(path: String).returns(Net::HTTP::Post) }
+      def post_request(path:)
+        request = Net::HTTP::Post.new(path)
+        request['X-SDK-VERSION'] = WorkOS::VERSION
```

Comment:
What do you think about just stuffing everything into the user agent? This is what Stripe/browser/HTTP-clients do

https://github.com/stripe/stripe-ruby/blob/master/lib/stripe/stripe_client.rb#L852

Reply:
@maxchehab wdyt about this? i think the goal is to make things more discoverable in logs - as long as the user agent string is searchable in logs I wouldn't mind it

Reply:
=I think we should just use the user agent header here. It will make it a little easier for filtering in the future if all versions are stored in the same header, including requests from the browser etc.

Reply:
kk back to the drawing board! 📝